src/model/torrent.rs

```diff
@@ -391,7 +391,7 @@ pub enum TorrentSource {
     /// URLs
     Urls { urls: Sep<Url, '\n'> },
     /// Raw data of torrent file.
-    TorrentFiles { torrents: Vec<u8> },
+    TorrentFiles { torrents: Vec<(String, Vec<u8>)> },
```

Comment:
Maybe we should make a dedicated struct for this. E.g. 

```
struct Torrent {
  filename: String,
  bytes: Vec<u8>
}
```

Reply:
Yes, I will add a
```Rust 
pub struct TorrentFile {
    pub filename: String,
    pub data: Vec<u8>
}
``` 
This helps to understand the parameters.